scripts/smp-server-droplet/marketplace-image.json

```diff
@@ -0,0 +1,53 @@
+{
+  "variables": {
+    "token": "{{env `DIGITALOCEAN_TOKEN`}}",
+    "image_name": "smp-server-20-04-snapshot-{{timestamp}}",
```

Comment:
can we either have `ubuntu-20-04` in image name, or no `20-04` at all - it's kind of ambiguous to what these numbers refer if you don't say `ubuntu` - first thing I'd think it's version of smp-server. I think just `smp-server` w/t `ubuntu-20-04` might be better, it's not like we're giving alternatives on digital ocean

Reply:
named `simplex-chat-server-snapshot`, better than abbreviations (this name will be seen by users if I'm not mistaken)